open-craft/opencraft
# BB-4728 Fix issue with Ocim marketing emails

## Description

This PR fixes a bug in the ocim marketing app where the P&T users are not being added to the Subscribers model once the first appserver is provisioned.

## Supporting Information

[BB-4728](https://tasks.opencraft.com/browse/BB-4728)

## Testing Instructions
- SSH into OCIM stage instance and attach to the stage screen session
- In the web window checkout to this branch `tinumide/fix_ocim_marketing_emails` in OCIM repository and do a `make run`
- Log  in to the ocim stage console at `https://stage.manage.opencraft.com/` and change `configuration_version` to `guruprasad/dummy-playbook` in current configuration settings
- Launch an app server
- Login to the OCIM stage admin and verify that the newly created user has been added to the `Subscribers` model
